macdmp.c

```diff
@@ -293,18 +397,59 @@ main (int argc, char **argv)
     case 'x':
       extract = 1;
       break;
+    case 'c':
+      input_word_format = &its_word_format;
```

Comment:
Does this need to set the word formats again? They haven't changed since the code above...

Reply:
The input/output formats are swapped.

Reply:
Ahh!